04_removeFromArray/README.md

```diff
@@ -10,7 +10,7 @@ removeFromArray([1, 2, 3, 4], 3); // should remove 3 and return [1,2,4]
 
 The first test on this one is fairly easy, but there are a few things to think about(or google) here for the later tests:
 
+- throughout the logic in your `removeFromArray` function, you can manipulate the original array you pass into the function call or create a new array that is returned as the result.
```

Comment:
I feel like the first part of the sentence is not needed, unless I am missing something.
```suggestion
- you can manipulate the original array you pass into the function call or create a new array that is returned as the result.
```

Reply:
Or, 
```suggestion
- when thinking about how your `removeFromArray` function will work, you might consider one of two approaches: 1. manipulate the original array that is passed into the function as an argument, or, 2. create a new array that includes only what should remain in the original array passed in.
```

Reply:
@fortypercenttitanium  I’m curious about how this initial hint goes over. If folks continue to struggle, we’ll add your additional context. Thanks for offering that! 🙏